ig/sushi-config.yaml

```diff
@@ -0,0 +1,244 @@
+# ╭─────────────────────────Commonly Used ImplementationGuide Properties───────────────────────────╮
+# │  The properties below are used to create the ImplementationGuide resource. The most commonly   │
+# │  used properties are included. For a list of all supported properties and their functions,     │
+# │  see: https://fshschool.org/docs/sushi/configuration/.                                         │
+# ╰────────────────────────────────────────────────────────────────────────────────────────────────╯
+id: cdc.gov.ti.ig
+canonical: http://cdc.gov/reportstream/trustedintermediary
+name: CDCTrustedIntermediaryIG
+# title: Example Title
+# description: Example Implementation Guide for getting started with SUSHI
+status: draft # draft | active | retired | unknown
+version: 0.1.0
+fhirVersion: 4.0.1 # https://www.hl7.org/fhir/valueset-FHIR-version.html
+copyrightYear: 2023+
+releaseLabel: ci-build # ci-build | draft | qa-preview | ballot | trial-use | release | update | normative+trial-use
+# license: CC0-1.0 # https://www.hl7.org/fhir/valueset-spdx-license.html
+# jurisdiction: urn:iso:std:iso:3166#US "United States of America" # https://www.hl7.org/fhir/valueset-jurisdiction.html
+publisher:
+  name: Flexion
+  url: http://flexion.us
+  # email: test@example.org
+
+# The dependencies property corresponds to IG.dependsOn. The key is the
+# package id and the value is the version (or dev/current). For advanced
+# use cases, the value can be an object with keys for id, uri, and version.
+#
+dependencies:
+  hl7.fhir.us.core: 6.0.0-ballot
```

Comment:
I don't think we should depend on the ballotted version until it comes out.  Our MVP is coming out before the ballotted version is published, so we shouldn't depend on it.

Reply:
We can change this to 5.0.1.  Looking through the changes in 6.0.0 there probably isn't anything that we need at the moment from the newer version.  However, the 6.0.0 changes to US Core reflect the ONC changes to USCDI v3 that were made in July of 2022.  Version 5 is from ONC changes to USCDI v2 that were made in July of 2021.

ONC releases new version updates to USCDI yearly in July.  They currently have USCDI v4 out for comment through April and they plan to publish v4 in July.  Changes will be made to the US Core FHIR profiles and balloted in January 2024 as US Core 7.0.0.

Reply:
Yeah, looks like we will work just fine with 5.0.1.  I recommend we stay with that until the new version is published.